github/ci/shared-deployments/files/docker-daemon-mirror.conf

```diff
@@ -1,5 +1,5 @@
 DOCKER_OPTS="${DOCKER_OPTS} --data-root=/docker-graph"
-DOCKER_OPTS="${DOCKER_OPTS} --registry-mirror=http://docker-mirror.kubevirt-prow.svc:5000 --insecure-registry=http://docker-mirror.kubevirt-prow.svc:5000"
```

Comment:
Please remove this change, this is our deployment code for our Prow control plane

Reply:
Done! Thanks!